src/cpp/session/modules/SessionThemes.R

```diff
@@ -1131,9 +1131,16 @@
       error = function(e) { stop("Unable to apply the theme \"", name, "\". ", e$message) })
 })
 
-# Remove a theme from RStudio.()
+# Remove a theme from RStudio.
 .rs.addApiFunction("removeTheme", function(name) {
    tryCatch(
       .rs.removeTheme(name, .Call("rs_getThemes")),
       error = function(e) { stop("Unable to remove the theme \"", name, "\". ", e$message) })
 })
+
+# Get the list of installed themes.
+.rs.addApiFunction("getThemes", function() {
+   lapply(.Call("rs_getThemes"), function(theme) {
```

Comment:
Nitpick: you should pass `PACKAGE = "(embedding)"` to `.Call()` as otherwise R will crawl all of the available loaded DLLs looking for a symbol called `rs_getThemes` and this can be slow if many DLLs (provided by R packages) are loaded.

Not a huge concern in this case but worth keeping in mind.